src/osd/osd_types.cc

```diff
@@ -5014,14 +5008,13 @@ void object_info_t::encode(bufferlist& bl, uint64_t features) const
   if (has_manifest()) {
     ::encode(manifest, bl);
   }
-  ::encode(extents, bl);
```

Comment:
::encode((uint32_t)0);

Reply:
and version 19. because 18 appears in mimic-dev1...

otherwise, we need to revert this (this patch only, i guess) before 13.0.1 is released.  maybe prepare a pr with just this branch?

Reply:
So final approach is to leave version 18 but I need to create a separate PR to remove 'extents' stuff from both master and mimic-dev1. Right?

Reply:
Yeah!  Thanks-